Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -15,6 +17,12 @@ import ShaderDestination from "../../Renderer/ShaderDestination.js";
 var GeometryPipelineStage = {};
 GeometryPipelineStage.name = "GeometryPipelineStage"; // Helps with debugging
 
+var attributesStructVSId = "ProcessedAttributesVS";
```

Comment:
hm yeah. Should we do the same with `.name -> .NAME`?

Reply:
The `name` property is fine, since it's only for debugging. With the shader builder struct functions, I think it's better if we do:

```
 shaderBuilder.addStructField(PROCESSED_ATTRIBUTES, "vec4", "color");
```

rather than

```
 shaderBuilder.addStructField(vsStructId, "vec4", "color");
```

Or just something along the lines to indicate that there is a global (-ish) struct that you are adding a field to.